### PR TITLE
Added MIT Micromasters graphic and updated title

### LINKED ISSUE #95: MIT Micromasters graphics
We should create graphics for the MIT Micromasters logo, see:
https://projects.invisionapp.com/share/KR6FYCZTX#/screens/142338434

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #95 

#### What's this PR do?
Uses the new MIT Micromasters graphic, and updates the title from "Micromasters" to "MIT Micromasters"

#### How should this be manually tested?
Open the dashboard and verify that you see a graphic with 'Micromasters' right next to MIT.

### CODE REVIEW

File: static/js/components/Header.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Header extends React.Component {
     return <div id="header">
       <div className="header-logo">
         <a href="/" alt="MIT Micromasters">
```
Comment (pdpinch) [inline static/js/components/Header.js]: No `alt` attribute on an `a` tag

File: static/js/components/Header.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ class Header extends React.Component {
...
-          <img src="/static/images/mit-white.png" alt="MIT"/>
+          <img src="/static/images/mit-micromasters.png" alt="MIT"/>
```
Comment (pdpinch) [inline static/js/components/Header.js]: Let's make this `alt="MIT"` to match the text of the graphc

File: static/js/components/Header.js (hunk 1 above)
Comment (noisecapella) [inline static/js/components/Header.js]: I don't think `a` tags should have `alt`: http://stackoverflow.com/questions/14850187/is-it-correct-to-use-alt-tag-for-an-anchor-link

**Comment by pdpinch** [comment]
:+1:

**Pull request merged**